.docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM php:8-fpm-alpine as runtime
+
+RUN apk update \
+  && apk add --no-cache curl make coreutils icu-libs icu-dev libzip-dev libxslt-dev php-mysqli php-pdo_mysql postgresql-dev libpng-dev
```

Comment:
@mmscondo thank you so much for the PR.

Just one mention, I see you are installing some dependencies that are not required at all for this project like `make`, `libpng-dev` or even `mysql` and `postgresql`.

I would remove them, what do you think?

Reply:
Yes I removed the extensions. I'll argue in the next PR why they existed.